packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/lib/create_saved_objects.ts

```diff
@@ -96,7 +105,12 @@ export const createSavedObjects = async <T>({
         ...(!importStateValue.omitOriginId && { originId: originId ?? object.id }),
       };
     }
-    return { ...object, ...(references && { references }), ...(originId && { originId }) };
+    return {
+      ...object,
+      ...(references && { references }),
+      ...(originId && { originId }),
+      ...{ managed: managed ?? object.managed ?? false }, // trictly speaking this shouldn't be needed since the objects passed in should already have the flag set
```

Comment:
Strictly speaking this shouldn't be needed since the objects passed in should already have the flag set